zerver/models.py

```diff
@@ -653,32 +654,40 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
 
     GIPHY_RATING_OPTIONS = {
         "disabled": {
-            "name": "GIPHY integration disabled",
+            "name": gettext_lazy("GIPHY integration disabled"),
             "id": 0,
         },
         # Source: https://github.com/Giphy/giphy-js/blob/master/packages/fetch-api/README.md#shared-options
         "y": {
-            "name": "Allow GIFs rated Y (Very young audience)",
+            "name": gettext_lazy("Allow GIFs rated Y (Very young audience)"),
             "id": 1,
         },
         "g": {
-            "name": "Allow GIFs rated G (General audience)",
+            "name": gettext_lazy("Allow GIFs rated G (General audience)"),
             "id": 2,
         },
         "pg": {
-            "name": "Allow GIFs rated PG (Parental guidance)",
+            "name": gettext_lazy("Allow GIFs rated PG (Parental guidance)"),
             "id": 3,
         },
         "pg-13": {
-            "name": "Allow GIFs rated PG13 (Parental guidance - under 13)",
+            "name": gettext_lazy("Allow GIFs rated PG13 (Parental guidance - under 13)"),
```

Comment:
Can you add an extra commit fixing the typo that should this be spelled `PG-13`?

Reply:
Sure!